async-aurora/Cargo.toml

```
@@ -0,0 +1,24 @@
+[package]
+name = "async_aurora"
+version = "0.1.0"
+edition = "2018"
+publish = false
+
+[lib]
+crate-type = ["cdylib", "rlib"]
+
+[profile.release]
+codegen-units = 1
+# Tell `rustc` to optimize for small code size.
+opt-level = "z"
+lto = true
+debug = false
+panic = "abort"
+overflow-checks = true
+
+[dependencies]
+near-sdk = { git = "https://github.com/near/near-sdk-rs", rev = "613adcab6322e4f830efb4d5bde2a810ae642158" }
```

The commit matches some random commit. Should be pointer to stable release.

```suggestion
near-sdk = { git = "https://github.com/near/near-sdk-rs", tag = "3.1.0" }
```